### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN003.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#49] [General]
# https://blog.3place.cn/3place/lmyn/LMYN003.html

https://blog.3place.cn/3place/lmyn/LMYN003.html

**Comment by picklecai** [comment]
一对麻鹬而被射死一只，这个故事和朱紫国的故事前因很像。

**Comment by picklecai** [comment]
从输迦到输洛迦，就好像从伤心到伤痕文学。
古代文体中，竟然难以找到这样演变的例子。

**Reply by picklecai** [reply]
各种词牌名，都是从最初的一件具体的事，演变到后来只代表某一套曲调韵律。

**Comment by picklecai** [comment]
只要那罗摩的故事，你所叙述的能流传，你也就能够永生，在天上，在人间。

作品流传，你就永生。和《寻梦环游记》一样，世上最后一个记得你的人也死了，你才真的烟消云散了。
物质没法永垂不朽，精神当然也没法，但精神可以传承。
创造多好，活着的时候，填充生命。死了之后，延长生命。
没有创造，人也不成为人了，谈不上灵了。

**Comment by picklecai** [comment]
用等量的音节和四个音步，大仙人把自己的悲痛抒发，由于翻来覆去地诉说吟咏，输迦于是就变成了输洛迦。

这是散文变韵文，这是普通情感变音乐。
蚁垤因为从生活而悟到了文学，他从普通人变成了“文人”，所以“思想净化、产生神智”。

**Comment by picklecai** [comment]
名词解释：
1. 牟尼：？
2. 达磨：？
